odoo/addons/base/res/res_config.py

```diff
@@ -470,6 +483,7 @@ def get_values(self):
     @api.model
     def default_get(self, fields):
         IrDefault = self.env['ir.default']
+        SuIrConfigParameter = self.env['ir.config_parameter'].sudo()
```

Comment:
Did you ever program in Smalltalk before? :stuck_out_tongue_winking_eye:

Reply:
Nope, why?